taxi/views.py

```diff
@@ -30,11 +37,24 @@
     return render(request, "taxi/index.html", context=context)
 
 
+# Manufacturer views----------------------------------------------------------
 class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     model = Manufacturer
     context_object_name = "manufacturer_list"
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
+    queryset = Manufacturer.objects.all()
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["search_query"] = self.request.GET.get("search_query", "")
+        return context
+
+    def get_queryset(self):
+        query = self.request.GET.get("search_query", "")
```

Comment:
The search parameter used here is `search_query`, but the `ManufacturerSearchForm` uses `search`. Ensure consistency between the form and view to make the search functionality work.